### DISCUSSION TITLE
Is the library fully compliant with ESP32 boards?

### DISCUSSION THREAD

**Discussion by slavino** [contrem/arduino-timer#41] [Q&A]
Hello,
is the lib fully compatible with ESP32 boards? I experience some weird behavior while using it on ESP32 and jut want to double check before I reach out to other solution.

Thanks a lot!

**Comment by Phozer** [comment]
I used the Library with the ESP32 in a LED Matrix Project. It worked for me and i didn't notice anything abnormal. I don't think thats alot of help, but i can't tell you anything else.

**Comment by contrem** [comment]
There's nothing hardware specific about this library, so it should work on any hardware without issue.

If you do find an issue, please share it here so we can try and address the cause.
